inference-engine/tests/functional/plugin/shared/include/behavior/set_blob.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "common_test_utils/data_utils.hpp"
+#include "common_test_utils/common_utils.hpp"
+
+namespace BehaviorTestsDefinitions {
+
+enum class setType {
+    INPUT,
+    OUTPUT,
+    BOTH
+};
+
+std::ostream& operator<<(std::ostream & os, setType type);
+
+using setBlobParams = std::tuple<InferenceEngine::Precision,   // input precision
+                                 InferenceEngine::Layout,      // input layout
+                                 InferenceEngine::Precision,   // output precision
+                                 InferenceEngine::Layout,      // output layout
+                                 setType,                      // type for which blob is set
+                                 InferenceEngine::SizeVector,  // input shape
+                                 std::string>;                 // Device name
+
+class setBlobTest : public testing::WithParamInterface<setBlobParams>, virtual public LayerTestsUtils::LayerTestsCommon {
```

Comment:
Why do you use `LayerTestsUtils::LayerTestsCommon`? It could be unjustifiably.
Furthermore use `SetBlobTest ` and `SetBlobParams ` instead of `setBlobTest ` and `setBlobParams `. These names are type names

Reply:
I believe we should compare result IE inference and NGrpah inference in this test